CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/ConfigurationPane.java

```diff
@@ -191,12 +188,11 @@ public void setSampleData(final String[] columnLabels, final List<String[]> curr
      */
     protected void clearSelectedPane() {
         if (tabPane.getSelectionModel().getSelectedItem() != null) {
-            RunPane currentSelected = (RunPane) tabPane.getSelectionModel().getSelectedItem().getContent();
+            final RunPane currentSelected = (RunPane) tabPane.getSelectionModel().getSelectedItem().getContent();
 
-            String[] columns = {};
-            List<String[]> data = new ArrayList<>();
+            final String[] columns = {};
 
-            currentSelected.setSampleData(columns, createTableRows(data));
+            currentSelected.setSampleData(columns, createTableRows(FXCollections.observableArrayList()));
```

Comment:
Sorry. When I meant replace the 2nd parameter, I meant replace the whole `createTableRows` function call. i.e.
`setSampleData(columns, FXCollections.observableArrayList());`